.gitignore

```diff
@@ -5,6 +5,15 @@ __pycache__
 **/install/*
 **/log/*
 
+# All playback subfolders
+**/bag_files/*

```

Comment:
I think should be `bag_files/**`. The `bag_files` directory should only appear in the root level and everything inside should be ignored, not just 1-level deep.